test/test_broker.hpp

```diff
@@ -714,7 +714,7 @@ class test_broker {
     bool unsubscribe_handler(
         Endpoint& ep,
         typename Endpoint::packet_id_t packet_id,
-        std::vector<mqtt::string_view> topics,
+        std::vector<mqtt::buffer> topics,
```

Comment:
There are several other places that need attention in this code as well.

E.g. see the "do_publish" function.

void do_publish(
        std::shared_ptr<std::string> const& topic,
        std::shared_ptr<std::string> const& contents,
        std::uint8_t qos,
        bool is_retain,
        std::vector<mqtt::v5::property_variant> props) {

Reply:
I believe that I fixed all part of test_broker using `mqtt::buffer`.
Please double check.